aten/src/ATen/native/native_functions.yaml

```diff
@@ -649,6 +653,9 @@
 - func: _baddbmm_mkl_(Tensor(a!) self, Tensor batch1, Tensor batch2, *, Scalar beta=1, Scalar alpha=1) -> Tensor(a!)
   variants: function
 
+- func: _sparse_mm_mkl_(Tensor(a!) self, Tensor sparse_, Tensor dense, Tensor t, Scalar alpha=1, Scalar beta=1) -> Tensor(a!)
+  variants: function
```

Comment:
you just gonna make me wonder why we have this function in native_functions.yaml at all

Reply:
Put it in a header file.